apix-impl/src/integration-test/java/eu/xenit/apix/tests/search/SearchServiceTest.java

```diff
@@ -154,18 +157,45 @@ public void TestSkip() throws IOException, InterruptedException {
 
     @Test
     public void TestTotalCount() throws IOException, InterruptedException {
+        transactionService.getRetryingTransactionHelper()
```

Comment:
Too bad we need to put sleeps here... Please add to scope of https://xenitsupport.jira.com/browse/ALFREDAPI-425 the location we need to clean up. So in this case, this sleep etc.